content/200-orm/200-prisma-client/100-queries/063-excluding-fields.mdx

```diff
@@ -4,15 +4,137 @@ metaTitle: 'Excluding fields'
 metaDescription: 'This page explains how to exclude sensitive fields from Prisma Client'
 ---
 
-<TopBlock>
-
 By default Prisma Client returns all fields from a model. You can use `select` to narrow the result set, but that can be unwieldy if you have a large model and you only want to exclude one or two fields.
 
-Prisma Client doesn't have a native way of excluding fields yet, but it's easy to create a function that you can use to exclude certain fields in a type-safe way.
+:::info
+
+As of Prisma ORM 5.16.0, excluding fields globally and locally is supported via the `omitApi` Preview feature.
+
+:::
+
+## Excluding a field globally using `omit`
+
+The following is a type-safe way to exclude a field globally using the `omitApi` Preview feature.
```

Comment:
```suggestion
The following is a type-safe way to exclude a field globally using the `omitApi` Preview feature:
```

Reply:
Maybe linking to what preview features are might be a good idea?
https://www.prisma.io/docs/orm/reference/preview-features